src/main/java/com/google/sps/servlets/RegistrationServlet.java

```diff
@@ -91,6 +91,8 @@ public void createStudentEntityAndPutInDatastore(DatastoreService ds, Entity ent
     entity.setProperty("pfp", pfp);
     entity.setProperty("email", email);
     entity.setProperty("learning", topics);
+    List<String> tutors = new ArrayList<String>();
+    entity.setProperty("tutors", tutors);
```

Comment:
I think adding an empty list to datastore will always add it as a null value. We can probably delete these lines and just set the property like you did in TutorSessionDatastoreService.

Reply:
That's true! It works